src/main/java/org/terasology/core/world/generator/facetProviders/SurfaceObjectProvider.java

```diff
@@ -40,7 +39,6 @@
  * for a environmental variable (e.g. biome).
  *
  */
-@Requires(@Facet(SurfaceHeightFacet.class))
```

Comment:
Is this gone because it was never required, or because it now depends on either `SurfaceHeightFacet` or `SurfacesFacet`?

Reply:
It seems to have been unnecessary in the first place. This class is abstract, so it's actually the annotations on its subclasses, the DefaultTreeProvider and DefaultFloraProvider, that matter.